submissions/antonkottans/hooli-style-popup/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<link rel="stylesheet" href="style.css">
+		<title>Popup window</title>
+	</head>
+	<body>
+		<header>
+			<nav>
+				<ul class="nav-menu">
+					<li class="nav-menu-item">
+						<a class="nav-menu-item-link" href="#">Mail</a>
+					</li>
+					<li class="nav-menu-item">
+						<a class="nav-menu-item-link" href="#">Images</a>
+					</li>
+					<li class="nav-menu-item">
+						<input
+							type="checkbox"
+							class="popup-checkbox hidden-checkbox"
+							id="popup-checkbox"
+						>
+						<label for="popup-checkbox" class="popup-checkbox-label">
+							<img class="nav-image" src="img/popup-button.png" alt="popup-menu" >
+						</label>
+						<div class="popup-menu">
+							<a href="#" class="popup-item"
+								><img
+									class="popup-item-img"
+									alt="calendar"
```

Comment:
For icons better set an empty `alt` attribute.
Sorry for russians but it's a good [article](https://htmlacademy.ru/blog/articles/alt-text)